weblate/accounts/models.py

```diff
@@ -496,6 +496,30 @@ class Profile(models.Model):
         default=False
     )
 
+    DASHBOARD_SUBSCRIPTIONS = "your-subscriptions"
+    DASHBOARD_LANGUAGES = "your-languages"
+    DASHBOARD_ALL = "projects"
+    DASHBOARD_COMPONENT_LIST = "list"
+    DASHBOARD_CHOICES = (
+        (DASHBOARD_SUBSCRIPTIONS, _('Your subscriptions')),
+        (DASHBOARD_LANGUAGES, _('Your languages')),
+        (DASHBOARD_ALL, _('All projects')),
+        (DASHBOARD_COMPONENT_LIST, _('Component list')),
+    )
+    dashboard_view = models.CharField(
+        max_length=100,
+        choices=DASHBOARD_CHOICES,
+        verbose_name=_('Default dashboard view'),
+        default=DASHBOARD_SUBSCRIPTIONS,
+    )
```

Comment:
Wouldn't be better to store these choices as integers rather than strings? We will anyway use symbolic names instead of them and storing integer in the database is cheaper than string...